dev/Tools/build/waf-1.7.13/lmbrwaflib/gems.py

```diff
@@ -619,7 +620,7 @@ def get_gem_by_spec(self, *spec):
     def contains_gem(self, *gem_spec):
         return self.get_gem_by_spec(*gem_spec) != None
 
-    def process(self):
+    def process(self, is_server):
```

Comment:
Maybe we can put default arg value for `is_server`:
`def process(self, is_server=False)`
in order to preserve backwards compatibility with older version?